tests/api/crud/test_alerts.py

```diff
@@ -21,66 +21,62 @@
 import sqlalchemy.orm
 
 import mlrun.common.schemas.alert
+import mlrun.common.schemas.alert as alert_objects
 import server.api.crud
 import tests.api.conftest
 
 
+@pytest.fixture
+def reset_alert(db, project="project-name", alert_name="failed-alert"):
```

Comment:
do you need this fixture? I believe the DB should start from scratch on each test

Reply:
it does not start from scratch.. without it only the first test pass and the others fail since the state of the alert is modified in the first test

Reply:
I think this fixture should also create the alert. You have duplicated the project and alert_name params and this fixture doesn't have any meaning if the alert was not created.
I'm still not sure why the DB doesn't start from scratch between tests since the fixture uses a tmp file as the DB file so that makes me think there might be some other issue here